### PR TITLE
ru: Fix note blocks

### PR DESCRIPTION
**Pull request by queengooborg**
This PR fixes up the note blocks for the Russian locale.  Due to inconsistencies in translation, note blocks were not converted properly, causing the Markdown to result in the equivalent of "Note: Note:" (or "Note: Info:", "Note: Callout:", "Note: Notes:", etc.).  This PR attempts to fix this.

Note: I am an upstream maintainer and do not speak Russian.  My changes aim to be purely editorial and technical.  Please feel free to suggest any changes that will improve the translation accuracy.

### CODE REVIEW

File: files/ru/web/css/animation-duration/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ animation-duration: 10s, 30s, 230ms;
 - `<time>`
   - : Длительность анимации определяется в секундах `s` или в миллисекундах `ms`. По умолчанию стоит значение `0s`. Отрицательные значения являются недействительными.
 
-> **Примечание:** **Предупреждение:** Отрицательные значения являются недействительными и отменяют объявление. Некоторые старые реализации считают отрицательные значения равнозначными `0s`.
+> **Предупреждение:** Отрицательные значения являются недействительными и отменяют объявление. Некоторые старые реализации считают отрицательные значения равнозначными `0s`.
```
Comment (Saionaro) [inline files/ru/web/css/animation-duration/index.md:32]:
```suggestion
> **Примечание:** Отрицательные значения являются недействительными и отменяют объявление. Некоторые старые реализации считают отрицательные значения равнозначными `0s`.
```

File: files/ru/web/css/background-size/index.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ translation_of: Web/CSS/background-size
 
 Значение **`background-size`** в [CSS](/ru/docs/CSS) позволяет задавать размер фонового изображения. Изображение может быть оставлено в исходном размере, растянуто, или подогнано под размеры доступного пространства.
 
-> **Примечание:** **Предупреждение:** Если значение этого свойства не задано в сокращённом свойстве {{cssxref("background")}}, которое применяется к элементу после CSS-свойства `background-size`, то значение этого свойства затем сбрасывается до исходного значения c помощью сокращённого свойства.
+> **Предупреждение:** Если значение этого свойства не задано в сокращённом свойстве {{cssxref("background")}}, которое применяется к элементу после CSS-свойства `background-size`, то значение этого свойства затем сбрасывается до исходного значения c помощью сокращённого свойства.
```
Comment (Saionaro) [inline files/ru/web/css/background-size/index.md:13]:
```suggestion
> **Примечание:** Если значение этого свойства не задано в сокращённом свойстве {{cssxref("background")}}, которое применяется к элементу после CSS-свойства `background-size`, то значение этого свойства затем сбрасывается до исходного значения c помощью сокращённого свойства.
```

File: files/ru/web/css/background/index.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ background: border-box red;
 background: no-repeat center/80% url("../img/image.png");
 ```
 
-> **Примечание:** **Предупреждение:** {{cssxref("background-color")}} можно определить только на последнем фоне, поскольку для всего элемента существует только один цвет фона.
+> **Предупреждение:** {{cssxref("background-color")}} можно определить только на последнем фоне, поскольку для всего элемента существует только один цвет фона.
```
Comment (Saionaro) [inline files/ru/web/css/background/index.md:30]:
```suggestion
> **Примечание:** {{cssxref("background-color")}} можно определить только на последнем фоне, поскольку для всего элемента существует только один цвет фона.
```

File: files/ru/web/css/border-image-source/index.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ translation_of: Web/CSS/border-image-source
 
 The **`border-image-source`** CSS property defines the {{cssxref("&lt;image&gt;")}} to use instead of the style of the border. If this property is set to `none`, the style defined by {{cssxref("border-style")}} is used instead.
 
-> **Примечание:** **Предупреждение:** Though any {{cssxref("&lt;image&gt;")}} can be used with this CSS property, browser support is still limited and some browsers support only images defined using the `url()` functional notation.
+> **Предупреждение:** Though any {{cssxref("&lt;image&gt;")}} can be used with this CSS property, browser support is still limited and some browsers support only images defined using the `url()` functional notation.
```
Comment (Saionaro) [inline files/ru/web/css/border-image-source/index.md:13]:
```suggestion
> **Примечание:** Though any {{cssxref("&lt;image&gt;")}} can be used with this CSS property, browser support is still limited and some browsers support only images defined using the `url()` functional notation.
```

File: files/ru/web/css/filter/index.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ translation_of: Web/CSS/filter
 
 В стандарт CSS включены несколько функций, которые обеспечивают предопределённые эффекты. Вы также можете ссылаться на SVG фильтр с URL-адресом на [элемент фильтра SVG](/ru/docs/Web/SVG/Element/filter).
 
-> **Примечание:** **Предупреждение:** Старые версии (с 4.0 до 9.0) браузера Windows Internet Explorer поддерживают нестандартный ["filter"](<http://msdn.microsoft.com/en-us/library/ms532853(v=vs.85).aspx>), который с тех пор устарел.
+> **Предупреждение:** Старые версии (с 4.0 до 9.0) браузера Windows Internet Explorer поддерживают нестандартный ["filter"](<http://msdn.microsoft.com/en-us/library/ms532853(v=vs.85).aspx>), который с тех пор устарел.
```
Comment (Saionaro) [inline files/ru/web/css/filter/index.md:22]:
```suggestion
> **Примечание:** Старые версии (с 4.0 до 9.0) браузера Windows Internet Explorer поддерживают нестандартный ["filter"](<http://msdn.microsoft.com/en-us/library/ms532853(v=vs.85).aspx>), который с тех пор устарел.
```

File: files/ru/web/media/formats/webrtc_codecs/index.md [hunk 6]
```diff
@@ -43,7 +43,7 @@ WebRTC устанавливает набор базовых кодеков, ко
 
 Полную информацию о том, какие видеокодеки и конфигурации требуется для поддержки WebRTC, можно найти в {{RFC (7742, "Требования к обработке видео и кодекам WebRTC")}}. Стоит отметить, что RFC охватывает множество требований, связанных с видео, включая цветовые пространства (sRGB является предпочтительным, но не обязательным цветовым пространством по умолчанию), рекомендации по функциям обработки веб-камеры (автоматическая фокусировка, автоматический баланс белого, автоматический уровень освещения) и так далее.
 
-> **Примечание:** **Предупреждение :** Эти требования относятся к веб-браузерам и другим продуктам, полностью совместимым с WebRTC. Продукты, не относящиеся к WebRTC, которые в некоторой степени могут взаимодействовать с WebRTC, могут поддерживать или не поддерживать эти кодеки, хотя это рекомендуется в технических документах
+> **Предупреждение:** Эти требования относятся к веб-браузерам и другим продуктам, полностью совместимым с WebRTC. Продукты, не относящиеся к WebRTC, которые в некоторой степени могут взаимодействовать с WebRTC, могут поддерживать или не поддерживать эти кодеки, хотя это рекомендуется в технических документах
```
Comment (Saionaro) [inline files/ru/web/media/formats/webrtc_codecs/index.md:46]:
```suggestion
> **Примечание:** Эти требования относятся к веб-браузерам и другим продуктам, полностью совместимым с WebRTC. Продукты, не относящиеся к WebRTC, которые в некоторой степени могут взаимодействовать с WebRTC, могут поддерживать или не поддерживать эти кодеки, хотя это рекомендуется в технических документах
```

File: files/ru/web/svg/attribute/d/index.md [hunk 7]
```diff
@@ -49,7 +49,7 @@ html,body,svg { height:100% }
 | Default value | _none_                                             |
 | Animatable    | Yes                                                |
 
-> **Примечание:** **Сноска:** Точка происхождения (координата `0`, `0`) обычно является верхним левым углом контекста. Однако элемент {{SVGElement("glyph")}} имеет своё происхождение в левом нижнем углу своего бокса.
+> **Сноска:** Точка происхождения (координата `0`, `0`) обычно является верхним левым углом контекста. Однако элемент {{SVGElement("glyph")}} имеет своё происхождение в левом нижнем углу своего бокса.
```
Comment (Saionaro) [inline files/ru/web/svg/attribute/d/index.md:52]:
```suggestion
> **Примечание:** Точка происхождения (координата `0`, `0`) обычно является верхним левым углом контекста. Однако элемент {{SVGElement("glyph")}} имеет своё происхождение в левом нижнем углу своего бокса.
```

File: files/ru/web/svg/attribute/stroke-dashoffset/index.md [hunk 8]
```diff
@@ -15,7 +15,7 @@ translation_of: Web/SVG/Attribute/stroke-dashoffset
 {{SVGRef}}
 Атрибут **`stroke-dashoffset`** определяет сдвиг массива **dash array** относительно начального положения.
 
-> **Примечание:** **Сноска:** Атрибут `stroke-dashoffset` может использоваться как свойство CSS.
+> **Сноска:** Атрибут `stroke-dashoffset` может использоваться как свойство CSS.
```
Comment (Saionaro) [inline files/ru/web/svg/attribute/stroke-dashoffset/index.md:18]:
```suggestion
> **Примечание:** Атрибут `stroke-dashoffset` может использоваться как свойство CSS.
```

File: files/ru/web/svg/attribute/stroke-linecap/index.md [hunk 9]
```diff
@@ -7,7 +7,7 @@ translation_of: Web/SVG/Attribute/stroke-linecap
 
 Атрибут **`stroke-linecap`** - это атрибут представления, определяющий форму, которая будет использоваться в конце открытых подпутей при штриховании.
 
-> **Примечание:** **Сноска:** В качестве атрибута представления можно использовать `stroke-linecap` в качестве свойства CSS.
+> **Сноска:** В качестве атрибута представления можно использовать `stroke-linecap` в качестве свойства CSS.
```
Comment (Saionaro) [inline files/ru/web/svg/attribute/stroke-linecap/index.md:10]:
```suggestion
> **Примечание:** В качестве атрибута представления можно использовать `stroke-linecap` в качестве свойства CSS.
```

File: files/ru/web/svg/element/pattern/index.md [hunk 10]
```diff
@@ -43,7 +43,7 @@ html,body,svg { height:100% }
   - : Этот атрибут определяет систему координат содержимого {{ SVGElement("pattern") }}.
     _Value type_: `userSpaceOnUse`|`objectBoundingBox`; _Default value_: `userSpaceOnUse`; _Animatable_: **yes**
 
-    > **Примечание:** **Сноска:** Этот атрибут не действует, если в элементе `<pattern>` указан атрибут `viewBox`.
+    > **Сноска:** Этот атрибут не действует, если в элементе `<pattern>` указан атрибут `viewBox`.
```
Comment (Saionaro) [inline files/ru/web/svg/element/pattern/index.md:46]:
```suggestion
    > **Примечание:** Этот атрибут не действует, если в элементе `<pattern>` указан атрибут `viewBox`.
```

File: files/ru/web/svg/element/pattern/index.md [hunk 11]
```diff
@@ -68,7 +68,7 @@ html,body,svg { height:100% }
   - : Этот атрибут ссылается на пример шаблона, предоставляющего значения по умолчанию для атрибутов `<pattern>`.
     _Value type_: [**\<URL>**](/docs/Web/SVG/Content_type#URL); _Default value_: _none_; _Animatable_: **yes**
 
-    > **Примечание:** **Сноска:** Для браузеров, реализующих `href`, если заданы как `href`, так и `xlink:href`, `xlink:href` будет игнорироваться, используя только `href`.
+    > **Сноска:** Для браузеров, реализующих `href`, если заданы как `href`, так и `xlink:href`, `xlink:href` будет игнорироваться, используя только `href`.
```
Comment (Saionaro) [inline files/ru/web/svg/element/pattern/index.md:71]:
```suggestion
    > **Примечание:** Для браузеров, реализующих `href`, если заданы как `href`, так и `xlink:href`, `xlink:href` будет игнорироваться, используя только `href`.
```

**Review by saionaro — APPROVED** [review]
LGTM, @queengooborg thank you!

File: files/ru/web/svg/tutorial/gradients/index.md [hunk 12]
```diff
@@ -58,7 +58,7 @@ translation_of: Web/SVG/Tutorial/Gradients
  <linearGradient id="Gradient2" x1="0" x2="0" y1="0" y2="1">
 ```
 
-> **Примечание:** **Замечание:** Вы также можете использовать атрибут `xlink:href` для градиентов. Когда он использован, атрибуты и стопы из одного градиента могут быть включены в другой. В примере выше можно было бы не пересоздавать все точки остановки в Gradient2**.**
+> **Примечание:** Вы также можете использовать атрибут `xlink:href` для градиентов. Когда он использован, атрибуты и стопы из одного градиента могут быть включены в другой. В примере выше можно было бы не пересоздавать все точки остановки в Gradient2**.**
```
Comment (queengooborg) [inline files/ru/web/svg/tutorial/gradients/index.md:61]:
```suggestion
> **Примечание:** Вы также можете использовать атрибут `xlink:href` для градиентов. Когда он использован, атрибуты и стопы из одного градиента могут быть включены в другой. В примере выше можно было бы не пересоздавать все точки остановки в Gradient2.
```

**Pull request merged**